setup.py

```diff
@@ -70,7 +70,7 @@
     },
     # Packaging
     package_dir={"": "src"},
-    packages=find_packages("src") + namespace_packages,
+    packages=find_namespace_packages(where="src"),
     # https://www.python.org/dev/peps/pep-0561/#packaging-type-information
     package_data={"": ["misc/*", "docs/*"], "inmanta": ["py.typed"]},
```

Comment:
I think this can go entirely.

The first part `"": ["misc/*", "docs/*"]` doesn't do anything
The second part is a duplicate of what is in Manifest.in